client/src/components/user/profile.jsx

```diff
@@ -1,10 +1,11 @@
 import React, { Component } from 'react';
 import { connect } from 'react-redux';
+import { Link } from 'react-router-dom';
 import NavigationBar from '../NavigationBar';
 import '../../styles/profile.scss';
 import getUserDetail from '../../actions/getUserDetail';
 import EditUserModal from '../Modal/EditUserModal';
-
+/* eslint-disable */
```

Comment:
### Nitpick

Just curious as to why or what you're disabling.
Also there should be a line break after your import statements and before defining your Class Component.